### PR TITLE
Release notes for 2.0.2

### PR DESCRIPTION
**Pull request by nacx**
Please, directly add your commits to the branch to complete/fix/amend the release notes.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://9ed730b115909ca859d5-7944c4141c2e5bc5b5d73aa31d838273.r19.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://f8931f7ae17f7b9e44d5-69377c7fc4b834cc42d973b7b7ec76b6.r82.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://87cc278c8f6a9bf24983-559fb9c9f81e64031f7b91d11a6460f9.r65.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://bfb899d8d21077de31ee-6ca9761b8aea880855d6076ac0974937.r91.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://bd97aa95d16e775204cf-4eb8e77b2b2b7ffb9525c5af5dcd3dcb.r95.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://f54d243c3b28f1e55ccf-03a253b2ad9712fc91cd302ec914599a.r78.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://d937201d592653f1b864-d01af2d2446f074b17b5efcbabdae6bf.r23.cf5.rackcdn.com/ to review your changes.

**Comment by trevorflanagan** [comment]
@nacx the DimensionData hyperlink is incorrect, should be - http://www2.dimensiondata.com
Only just spotted now.

**Comment by trevorflanagan** [comment]
Sorry @nacx ignore my last comment.

**Comment by nacx** [comment]
Merged and published!

File: releasenotes/2.0.2.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* [JCLOUDS-1307](https://issues.apache.org/jira/browse/JCLOUDS-1307) - Security group can't be re-created if deleted externally
+* [JCLOUDS-1315](https://issues.apache.org/jira/browse/JCLOUDS-1315) - Add InternetGatewayApi to AWSEC2Api
+* [JCLOUDS-1316](https://issues.apache.org/jira/browse/JCLOUDS-1316) - Add a RouteTableApi to AWSEC2Api
+* [JCLOUDS-1317](https://issues.apache.org/jira/browse/JCLOUDS-1317) - Add ModifySubnetAttribute to AWSSubnetApi
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12339744&styleName=Html&projectId=12314430).
+
+## <a id="knownissues"></a> Known Issues
+
+TODO
```
Comment (demobox) [inline releasenotes/2.0.2.md]: "None"? ;-)

**Pull request closed without merging**